eos/constraint.cc

```diff
@@ -1311,7 +1294,7 @@ namespace eos
         {
```

Comment:
Looking at line 1289 (1306 original) above, i wonder if ``dof`` should now only be emitted if ``dim_meas != dim_pred``. Or could we simply use ``dof == dim_pred`` always?

Reply:
No, Christoph sent me another example where `dof < dim_meas = dim_pred` https://github.com/eos/eos/issues/477#issuecomment-1170368309. I think that the user may want to use `dof` to modify the test statistics easily. This is also something we may want to use when implementing a combination of two independent measurements, no?

Reply:
Very good, I'm happy.